token-leak-process.md

```diff
@@ -0,0 +1,33 @@
+# GitHub Token Leak Response Process
+
+The SRC may become aware that a Kubernetes Org member has leaked a GitHub token. Because Org membership can confer privileges, it is important to ensure leaked GitHub tokens are revoked. This process describes how to do so.
+
+## Investigate
+
+* Thank the reporter -- they have done us a kindness by letting us know about the problem
+* If you can, get the first few characters of the leaked token in case GitHub support needs it for revocation
+* Check to see if the user is an OWNER - an OWNER token leak would be more severe than that of an ordinary Org member
+    * https://cs.k8s.io/?i=fosho&files=OWNERS&q=the-user-name-goes-here
+
+## Contact
+
+* Contact the user via email, letting them know about the token leak
+>Dear USER,
```

Comment:
nit: consider bolding or using code blocks to make the things that need replacing pop more:

>Dear **USER**,

or

>Dear `USER`,

Reply:
Done!